packages/unmagic/unmagic.1.0.3/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+version: "1.0.3"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+homepage: "https://bitbucket.org/camlspotter/unmagic/"
+bug-reports: "https://bitbucket.org/camlspotter/unmagic/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/unmagic"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build & < "0.10" }
+  "spotlib"
+  "typerep"
+  "ppx_typerep_conv" { build }
+  "ppx_deriving"
+]
+available: [
+  ocaml-version >= "4.06.0"
```

Comment:
I'm curious. Where does this restriction come from ?

Reply:
RIght, it does not require the ocaml-version constraint.  Thanks!